### PR TITLE
Create feature toggle to turn on/off autocomplete for input fields

### PR DESCRIPTION
**Pull request by ivankanag**

### CODE REVIEW

**Comment by akgagnat** [comment]
Legg inn enabled-features: "autoComplete" i dev.yaml og dev-forhandsvisning.yaml

**Review by akgagnat — APPROVED** [review]
:goal_net: :soccer: :goat: :dash:

File: packages/shared-components/src/Forms/components/ModalPrompt.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { Normaltekst, Systemtittel } from "nav-frontend-typografi";
+import React, { useEffect, useState } from "react";
+import { useLanguages } from "../../context/languages/languages-context";
+
+const ModalPrompt = () => {
+  const { translate } = useLanguages();
+  const [openModal, setOpenModal] = useState(false);
+
+  useEffect(() => {
+    console.log("modal", openModal);
```
Comment (magnurh-cx) [inline packages/shared-components/src/Forms/components/ModalPrompt.jsx:13]: fjern hele filen

**Review by magnurh-cx — APPROVED** [review]
🐐 🗡️ 🎈

**Pull request merged**